### PR TITLE
Initial commit for model as described in 'Character-Level CNNs for Te…

### PR DESCRIPTION
**Pull request by renwang435**
…xt Classification (Zhang et al. 2016)'

Character level model which achieves 58.87% validation accuracy on the one-of-five ratings classification problem after 60 epochs, trained on 10000 reviews. Architecture described in model.py. Hyperparameters described in config.json.

### CODE REVIEW

**Review by iKevinY — APPROVED** [review]
LGMT! I like how the model is specified in a JSON file for easy tweaking 👍
